types.nix

```diff
@@ -606,7 +606,7 @@ rec {
         readOnly = true;
         type = types.functionTo (types.listOf types.package);
         default = pkgs:
-          [ pkgs.parted pkgs.systemdMinimal ] ++ flatten (map (partition: partition._pkgs pkgs) config.partitions);
+          [ pkgs.parted pkgs.systemdMinimal pkgs.lvm2 pkgs.gnugrep pkgs.coreutils pkgs.findutils ] ++ flatten (map (partition: partition._pkgs pkgs) config.partitions);
```

Comment:
Could be lvm2 moved to lvm partitions?

Reply:
We need `blkdeactivate` from lvm2 package even if we don't use lvm

Reply:
I see. Maybe add a comment why this is needed.